app/src/containers/b2b/AccountMain.tsx

```diff
@@ -0,0 +1,37 @@
+
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import * as React from 'react';
+
+
+
+export default class AccountMain extends React.Component {
+    constructor(props: any) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div><h1>Account</h1></div>
```

Comment:
"Account" should not be a hardcoded string

Reply:
> "Account" should not be a hardcoded string

Yes, of course, we're working on this component now